client/zweb/deployer/index.js

```diff
@@ -3,11 +3,16 @@ const fs = require('fs');
 const utils = require('#utils');
 const ethUtil = require('ethereumjs-util');
 
+// TODO: direct import cause fails in some docker scripts
+let storage;
+
 class Deployer {
     constructor(ctx) {
         this.ctx = ctx;
         this.config = this.ctx.config.deployer;
         this.cache_uploaded = {}; // todo: unused? either remove or use
+        const _storage = require("../../../client/storage/index-new.js");
```

Comment:
As long as you have removed both `index.js` and `index-arweave.js` let's rename the file to just `index.js`.

Reply:
This is done for easier reviewing, otherwise you would see the diff between the old and new files. I am planning to rename it as a next step